Makefile

```diff
@@ -114,17 +133,19 @@ docker-push:
 	docker push ${DOCKER_TAG}
 
 .openshift-validate:
-	@oc get project ${NAMESPACE}
+	@oc get project ${NAMESPACE} > /dev/null
 
 openshift-deploy: openshift-undeploy
 	@echo Deploying to OpenShift project ${NAMESPACE}
-	oc create -f deploy/openshift/sws-configmap.yaml -n ${NAMESPACE}
-	oc process -f deploy/openshift/sws.yaml -p IMAGE_NAME=${DOCKER_NAME} -p IMAGE_VERSION=${DOCKER_VERSION} -p NAMESPACE=${NAMESPACE} | oc create -n ${NAMESPACE} -f -
+	oc process -f deploy/openshift/sws.yaml -p IMAGE_NAME=${DOCKER_NAME} -p IMAGE_VERSION=${DOCKER_VERSION} -p NAMESPACE=${NAMESPACE} -p SERVICE_PORT=${SERVICE_PORT} -p VERBOSE_MODE=${VERBOSE_MODE} | oc create -n ${NAMESPACE} -f -
 
 openshift-undeploy: .openshift-validate
 	@echo Undeploying from OpenShift project ${NAMESPACE}
 	oc delete all,secrets,sa,templates,configmaps,daemonsets,clusterroles,clusterrolebindings --selector=app=sws -n ${NAMESPACE}
 
+openshift-reload-image: .openshift-validate
+	@oc delete $$(oc get pods -l app=sws -o name)
```

Comment:
A couple of comments here:
This command is incorrect. You need to specify the namespace, otherwise if the user is logged into oc using another namespace, it wont work.

You can also simplify this command, is should be just:
`oc -n istio-system delete pod --selector=app=sws`

And its bugging me why we need a make option for a single line command. We are dealing with OpenShift and Istio, developers should be familiar with how to do things in OpenShift/K8s.